CMakeLists.txt

```
@@ -345,15 +337,23 @@ if(Launcher_QT_VERSION_MAJOR EQUAL 6)
     set(QT_LIBEXECS_DIR ${QT${QT_VERSION_MAJOR}_INSTALL_PREFIX}/${QT${QT_VERSION_MAJOR}_INSTALL_LIBEXECS})
 endif()
 
+find_package(PkgConfig REQUIRED)
```

```suggestion
find_package(PkgConfig)
```
This technically isn't required since we have a fallback